client/components/jetpack/connection-health/index.tsx

```diff
@@ -45,6 +56,75 @@ export const JetpackConnectionHealthBanner = ( { siteId }: Props ) => {
 		);
 	}
 
+	if ( errorType === FATAL_ERROR ) {
+		return (
+			<ErrorNotice
+				errorType={ errorType }
+				errorText={ translate(
+					"Jetpack is unable to communicate with your site due to a critical error that has occurred there. Please check your site admin's email inbox for instructions."
+				) }
+				noticeActionHref={ localizeUrl(
+					'https://wordpress.com/support/why-is-my-site-down/#theres-a-critical-error-on-your-site'
+				) }
+				noticeActionText={ translate( 'Learn how to fix' ) }
+			/>
+		);
+	}
+
+	if ( [ USER_TOKEN_ERROR, BLOG_TOKEN_ERROR ].includes( errorType ) ) {
+		return (
+			<ErrorNotice
+				errorType={ errorType }
+				errorText={ translate(
+					'Jetpack is unable to communicate with your site due to a token error. Please reconnect Jetpack.'
```

Comment:
```suggestion
					'Jetpack is unable to communicate with your site because your site isn’t connected.'
```

I think 'token error' falls into the category of '[technical jargon](https://twitter.com/carlvellotti/status/1688714484175589376/photo/1)'. Also, 'Please reconnect Jetpack' conflicts with the notice action text.

Reply:
+1 but instead of `Jetpack is unable to communicate` I'd say `Jetpack can't communicate`